### DISCUSSION TITLE
Save Path Blank

### DISCUSSION THREAD

**Discussion by gregeeh** [Novik/ruTorrent#2290] [Q&A] [answered]
Hi all,

If I download a torrent the path it was saved to is shown in the Save Path column in rutorrent.  If you restart rutorrent it then goes blank.
Is this considered "normal" behaviour and if so what's the logic behind blanking it after a restart?

TIA

**Comment by Novik** [comment, marked as answer]
> Is this considered "normal" behaviour

Yes. 
rtorrent doesn't know saving paths for closed torrents. You should start torrent as minimum once per rtorrent session.

**Reply by gregeeh** [reply]
Thank you, understood.
